lib/cloudwatch-integration.js

```diff
@@ -129,22 +129,38 @@ lib.getToken = function(aws, groupName, streamName, cb) {
   });
 };
 
-lib.ensureGroupPresent = function ensureGroupPresent(aws, name, cb) {
+lib.ensureGroupPresent = function ensureGroupPresent(aws, name, retentionInDays, cb) {
   debug('ensure group present');
   var params = { logGroupName: name };
   aws.describeLogStreams(params, function(err, data) {
     // TODO we should cb(err, false) if there's an error?
     if (err && err.code == 'ResourceNotFoundException') {
       debug('create group');
       return aws.createLogGroup(params, lib.ignoreInProgress(function(err) {
+        if(!err) lib.putRetentionPolicy(aws, name, retentionInDays);
         cb(err, err ? false : true);
       }));
     } else {
+      lib.putRetentionPolicy(aws, name, retentionInDays);
       cb(err, true);
     }
   });
 };
 
+lib.putRetentionPolicy = function putRetentionPolicy(aws, groupName, days) {
+  var params = {
+    logGroupName: groupName,
+    retentionInDays: days
+  };
+  if(days > 0) {
+    aws.putRetentionPolicy(params, function(err, data) {
+      if (err) console.log(err, err.stack);
+      else     console.log(data);
+      // TODO - not sure if we care here?
```

Comment:
I think we don't care about the positive case, unless there's some actionable content in `data` that we need to use later on. 

In the negative case I am happy as you did, I don't think we should `cb(err)` if the retention policy did not succeed.

Could you please add `debug('setting retention policy for ' + groupName + ' to ' + days + ' days');` after line 150 so we could trace that too in case we will need it?